lib/fastlane/plugin/wpmreleasetoolkit/helper/po_extended.rb

```diff
@@ -0,0 +1,24 @@
+require 'gettext/po'
+
+## Extend GetText::PO to allow sorting by msgctxt
+class PoExtended < GetText::PO
```

Comment:
is `PoExtended` a good name for this? 
Is this folder a good place to keep such a class?

Reply:
Indeed it was very strange that that sorting was not supported. Like you say we must be using it in a `fancy` way else it would have been supported.

For the future I'd love to investigate `GlotPress` to see if a proper usage could be adopted. For the sake of this PT though, I just produced what we already have :)